api/HashChangeEvent.json

```diff
@@ -121,22 +121,22 @@
               "version_added": false
             },
             "opera": {
-              "version_added": true
+              "version_added": "≤12.1"
             },
             "opera_android": {
-              "version_added": true
+              "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": true
+              "version_added": "5.1"
```

Comment:
This also doesn't quite match `onhashchange` data. Can you make bring them into alignment?